Odata-docs/webapi/pagesize-top-maxtop.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "PageSize, Top and MaxTop"
+description: "This tutorial describes how to configure PageSize, Top and MaxTop."
+
+author: KenitoInc
+ms.author: kemunga
+
+ms.date: 3/1/2022
+---
+# PageSize, Top and MaxTop
+**Applies To**:[!INCLUDE[appliesto-webapi](../includes/appliesto-webapi-v7.md)][!INCLUDE[appliesto-webapi](../includes/appliesto-webapi-v6.md)]
+
+In OData WebApi we support both client driven and client side paginations.
+
+### Client driven pagination
+In client-driven paging, we request the server to return the specified number of results. There is no `nextLink` that is returned.
+
+The OData WebApi deals with client-driven paging using `$skip` and `$top` query options.
+
+The `$top` query option requests the number of items in the queried collection to be included in the result.
+
+The `$skip` query option requests the number of items in the queried collection that are to be skipped and not included in the result.
+
+The `MaxTop()` sets the maximum value that can be set for the `$top` query parameter. It does not directly limit the number of items returned per request.
+
+In `Microsoft.AspNetCore.OData v7.x`, we configure `MaxTop()` as follows:
```

Comment:
What about v8.x?

Reply:
This doc page is in the v7.x section. We can fix this when we add v8.x docs, or when we have a common section for both 7.x and 8.cx docs